shared/chat/conversation/attachment-fullscreen/container.js

```diff
@@ -45,7 +47,7 @@ const mapDispatchToProps = (dispatch, {navigateUp, navigateAppend}: OwnProps) =>
       dispatch(FsGen.createOpenLocalPathInSystemFileManager({localPath: message.downloadPath}))
   },
   onClose: () => {
-    dispatch(navigateUp())
+    dispatch(flags.useNewRouter ? RouteTreeGen.createNavigateUp() : navigateUp())
```

Comment:
just fyi i think its safe to just dispatch the global one and ignore the local one. nav1 will go away so im' not worried about losing the actionifonpath version